cluster-autoscaler/hack/update-vendor.sh

```diff
@@ -17,7 +17,7 @@ fi
 
 SCRIPT_NAME=$(basename "$0")
 K8S_FORK="git@github.com:kubernetes/kubernetes.git"
-K8S_REV="master"
+K8S_REV="release-1.16"
```

Comment:
This can be overrided from argument. You don't need to make this change.

Reply:
I did this to replicate the work already done in the 1.15 branch (see #2971). This is a PR against the 1.16 branch, not master.